Mathlib/Data/Real/Basic.lean

```diff
@@ -246,12 +248,11 @@ def ringEquivCauchy : ℝ ≃+* CauSeq.Completion.CauchyCat abs :=
  `field ℝ` is found first, then decaying it to these typeclasses would result in a `noncomputable`
  version of them. -/
 
-
 instance : Ring ℝ := by infer_instance
 
 instance : CommSemiring ℝ := by infer_instance
 
-instance : Semiring ℝ := by infer_instance
+instance semiring : Semiring ℝ := by infer_instance
```

Comment:
Is this intentional? The other instances are not named.

Reply:
It's used by name in the `StrictOrderedCommRing` instance